api/src/DataPersister/ContentNode/MultiSelectDataPersister.php

```diff
@@ -19,23 +19,22 @@ public function __construct(
         );
     }
 
+    /**
+     * @param MultiSelect $data
+     */
     public function beforeCreate($data): MultiSelect {
-        /** @var MultiSelect $multiSelect */
-        $multiSelect = $data;
-
-        if (isset($multiSelect->prototype)) {
-            // copy from Prototype
+        parent::beforeCreate($data);
```

Comment:
ich fände es schöner, mit dem rückgabewert von parent::beforeCreate($data) weiterzufahren.
Sonst muss man sich darauf verlassen, dass änderungen in place passieren

Reply:
done